src/scheduler/server_info.cpp

```diff
@@ -460,6 +461,34 @@ query_server(status *pol, int pbs_sd)
 	 * we don't want to account for resources consumed by ghost jobs
 	 */
 	create_placement_sets(policy, sinfo);
+	if (!sinfo->node_group_enable && sinfo->node_group_key != NULL &&
+			strcmp(sinfo->node_group_key[0], "msvr_node_group") == 0) {
```

Comment:
This might be a dumb question, "msvr_node_group" sounds very generic to me. What happens if the admin sets the same name to one of his placement sets?

Also, you might want to make it a #define or a const variable since you used it in more than one place.

Reply:
Hmm .. admins set pset to resource names that they can set on the nodes .. msvr_node_group is a resource .. so they won’t be able to create another resource with the same name .. msvr_node_group is also not settable, so even if they read about it in the guides and tried to use it, they won’t be able to .. so i think it should be okay .. let me know what you think

I should be able to use 'RESC_MSVR_ND_GROUP' instead of the string literal, I'll change it thanks